src/tlo/methods/rti.py

```diff
@@ -4094,25 +4178,19 @@ def apply(self, person_id, squeeze_factor):
 
         # If they have an open fracture, ask for consumables to treat fracture
         if open_fracture_counts > 0:
-            self.module.item_codes_for_consumables_required['open_fracture_treatment'] = {
-                get_item_code('Ceftriaxone 1g, PFR_each_CMST'): 2000,
-                get_item_code('Cetrimide 15% + chlorhexidine 1.5% solution.for dilution _5_CMST'): 500,
-                get_item_code("Gauze, absorbent 90cm x 40m_each_CMST"): 100,
-                get_item_code('Suture pack'): 1,
-            }
             # If wound is "grossly contaminated" administer Metronidazole
             # todo: parameterise the probability of wound contamination
             p = self.module.parameters
             prob_open_fracture_contaminated = p['prob_open_fracture_contaminated']
             rand_for_contamination = self.module.rng.random_sample(size=1)
             # NB: Dose used below from BNF is for surgical prophylaxsis
             if rand_for_contamination < prob_open_fracture_contaminated:
-                self.module.item_codes_for_consumables_required['open_fracture_treatment'].update(
+                self.module.cons_item_codes['open_fracture_treatment'].update(
```

Comment:
Same comment as above about us probably not wanting to edit the module's store of the consumables